Lib/test/test_csv.py

```diff
@@ -1018,6 +1019,16 @@ def test_doublequote(self):
         dialect = sniffer.sniff(self.sample9)
         self.assertTrue(dialect.doublequote)
 
+    def test_guess_lineterminator(self):
+        sniffer = csv.Sniffer()
+        dialect = sniffer.sniff(self.sample5)
+        self.assertEqual(dialect.lineterminator, '\r\n')
+        dialect = sniffer.sniff(self.sample8)
+        self.assertEqual(dialect.lineterminator, '\n')
```

Comment:
This assumes that the line terminator for multiline triple-quoted string literals is '\n'.  We require that '\r\n' in code files be converted to '\n' before merging, but the test should not assume '\n' for all developer and user systems where the test might be run.  So `sample8` should be replaced by a literal.

Since the sample strings need not be long, they can all be literals defined here.  Make all 3 the same except for the terminator.  This will make the test both crystal clear and independent of system and optional local settings.